spec/integration_spec.rb

```diff
@@ -25,7 +25,8 @@ def build_organizer(options = {}, &block)
   #  └─ interactor5
 
   let(:organizer) {
-    build_organizer(organize: [organizer2, interactor3, organizer4, interactor5]) do
+    interactors = [organizer2, interactor3, organizer4, interactor5]
+    build_organizer(organize: interactors) do
```

Comment:
Yeah, I'm all for the 80 character limit but I typically relax that rule a bit for specs. But maybe I shouldn't!